src/ansys/mapdl/core/mapdl_core.py

```diff
@@ -235,7 +235,6 @@ def __init__(
     ):
         """Initialize connection with MAPDL."""
         atexit.register(self.__del__)  # registering to exit properly
-        self._name = None  # For naming the instance.
```

Comment:
I dont see the reason to move `name` out of the main class. Why don't you want to have it in `MapdlInprocess`? Some errors might use it for printing their messages.

Reply:
`name` has not been moved, I just removed `self._name`